core/src/main/java/io/aiven/klaw/config/ManageDatabase.java

```diff
@@ -112,7 +112,7 @@ public class ManageDatabase implements ApplicationContextAware, InitializingBean
   @Value("${klaw.admin.mailid}")
   private String kwAdminMailId;
 
-  @Value("${klaw.superadmin.defaultpassword}")
+  @Value("${klaw.superadmin.default.password:'changethepassword'}")
```

Comment:
Is this safe? Didn't most of the PHP forum sites had this problem? Admins didn't know better and were using the default password making it easy to exploit?

Reply:
I was thinking the same, but now aligned it with username, checking for blank.